rotkehlchen/chain/ethereum/modules/diva/decoder.py

```diff
@@ -0,0 +1,136 @@
+from rotkehlchen.accounting.structures.balance import Balance
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.chain.ethereum.utils import token_normalized_value
+from rotkehlchen.chain.evm.decoding.interfaces import DecoderInterface
+from rotkehlchen.chain.evm.decoding.structures import (
+    DEFAULT_DECODING_OUTPUT,
+    DecoderContext,
+    DecodingOutput,
+)
+from rotkehlchen.chain.evm.decoding.types import CounterpartyDetails, EventCategory
+from rotkehlchen.chain.evm.structures import EvmTxReceiptLog
+from rotkehlchen.chain.evm.types import string_to_evm_address
+from rotkehlchen.constants.assets import A_DIVA
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+from rotkehlchen.types import ChecksumEvmAddress, DecoderEventMappingType, EvmTransaction
+from rotkehlchen.utils.misc import hex_or_bytes_to_address, hex_or_bytes_to_int
+
+from .constants import CPT_DIVA, DIVA_ADDRESS
+
+if TYPE_CHECKING:
+    from rotkehlchen.chain.evm.decoding.base import BaseDecoderTools
+    from rotkehlchen.chain.evm.node_inquirer import EvmNodeInquirer
+    from rotkehlchen.user_messages import MessagesAggregator
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+DELEGATE_CHANGED = b'14\xe8\xa2\xe6\xd9~\x92\x9a~T\x01\x1e\xa5H]}\x19m\xd5\xf0\xbaMN\xf9X\x03\xe8\xe3\xfc%\x7f'  # noqa: E501
+CLAIM_AIRDROP = b'N\xc9\x0e\x96U\x19\xd9&\x81&tg\xf7u\xad\xa5\xbd!J\xa9,\r\xc9=\x90\xa5\xe8\x80\xce\x9e\xd0&'  # noqa: E501
+DIVA_AIDROP_CONTRACT = string_to_evm_address('0x777E2B2Cc7980A6bAC92910B95269895EEf0d2E8')
+
+
+class DivaDecoder(DecoderInterface):
+
+    def __init__(
+            self,
+            evm_inquirer: 'EvmNodeInquirer',
+            base_tools: 'BaseDecoderTools',
+            msg_aggregator: 'MessagesAggregator',
+    ) -> None:
+        super().__init__(
+            evm_inquirer=evm_inquirer,
+            base_tools=base_tools,
+            msg_aggregator=msg_aggregator,
+        )
+        self.diva = A_DIVA.resolve_to_evm_token()
+
+    def _process_delegation_change(
+            self,
+            topics: list[bytes],
+            transaction: EvmTransaction,
+            tx_log: EvmTxReceiptLog,
+    ) -> DecodingOutput:
+        """Process information for changes in delegation"""
+        delegator = hex_or_bytes_to_address(topics[1])
+        delegate = hex_or_bytes_to_address(topics[3])
+
+        if self.base.is_tracked(delegator):
+            event_address = delegator
+        else:
+            event_address = delegate
+
+        event = self.base.make_event_from_transaction(
+            transaction=transaction,
+            tx_log=tx_log,
+            event_type=HistoryEventType.INFORMATIONAL,
+            event_subtype=HistoryEventSubType.GOVERNANCE,
+            asset=self.diva,
+            balance=Balance(),
+            location_label=event_address,
+            notes=f'{delegator} started to delegate on {self.diva.symbol} to {delegate}',
```

Comment:
Change the notes to use exactly same format as https://github.com/rotki/rotki/blob/8ff5b44bc4ff1940bf22cc6b80b2e486ceaf2a66/rotkehlchen/chain/optimism/modules/optimism/decoder.py#L47

We need to standardize all these. Both airdrop claims and delegations

Also don't make diva.symbol a variable. It's constant.